ci/build_container/build_container_common.sh

```diff
@@ -24,3 +24,10 @@ export THIRDPARTY_BUILD=/thirdparty_build
 export CPPFLAGS="-DNDEBUG"
 echo "Building opt deps ${DEPS}"
 "$(dirname "$0")"/build_and_install_deps.sh ${DEPS}
+
+echo "Building Bazel-managed deps (//bazel/external:all_external)"
+BAZEL_OPTIONS="--output_user_root=/bazel-prebuilt-root --output_base=/bazel-prebuilt-output"
+cd /bazel-prebuilt
+for BAZEL_MODE in opt dbg fastbuild; do
+  bazel ${BAZEL_OPTIONS} build -c "${BAZEL_MODE}" //bazel/external:all_external
+done
```

Comment:
@jmillikin-stripe we should make sure we switch the main Envoy build (and CI) to the new image that results from this change. Can you do a PR to bump https://github.com/envoyproxy/envoy/blob/master/ci/envoy_build_sha.sh? Thanks.

Reply:
Mailed https://github.com/envoyproxy/envoy/pull/1867